src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -89,6 +101,13 @@ class ResourceLocator extends React.Component<FieldTypeProps<?string>> {
         onFinish();
     };
 
+    handleButtonClick = () => {
```

Comment:
Not sure about the naming yet - I would prefer "Regenerate URL" over "Refresh URL") - but depending on what we choose I would call this method `handleRegenerateButtonClick` or `handleRefreshButtonClick`.

Reply:
I renamed it to handleRegenerateButtonClick for now, since generate is used within Sulu related to the URL.